apps/spruce/src/pages/task/taskTabs/TaskHistory/CommitDetailsList/index.tsx

```diff
@@ -3,17 +3,20 @@ import { ParagraphSkeleton } from "@leafygreen-ui/skeleton-loader";
 import { size } from "@evg-ui/lib/constants/tokens";
 import { TaskQuery } from "gql/generated/types";
 import CommitDetailsCard from "../CommitDetailsCard";
+import InactiveCommitsButton from "../InactiveCommitsButton";
 import { GroupedTask } from "../types";
 
 interface CommitDetailsListProps {
   currentTask: NonNullable<TaskQuery["task"]>;
   tasks: GroupedTask[];
   loading: boolean;
+  shouldCollapse: boolean;
 }
 
 const CommitDetailsList: React.FC<CommitDetailsListProps> = ({
   currentTask,
   loading,
+  shouldCollapse,
```

Comment:
(after above change)
```suggestion
}

const CommitDetailsList: React.FC<CommitDetailsListProps> = ({
  currentTask,
  loading,
```